core/tests/puppeteer-acceptance-tests/blogpostAdmin-tests/blog-post-create-draft.js

```diff
@@ -0,0 +1,19 @@
+const e2eBlogPostAdmin = require("../utility-functions/blogPostAdminUtils.js");
+const testConstants = require("../utility-functions/testConstants.js");
```

Comment:
A "test constant" is not a utility function (since constants are not functions). Could you think through the naming here a bit more? What are "test constants", exactly?

Perhaps utility-functions needs to get renamed to testing-utilities, or similar. But that is really general so it would be better to find a more specific name that describes more clearly what it contains (e.g. are they puppeteer-specific test utils, jasmine-specific test utils, etc.?).